MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -169,54 +169,48 @@ object VisionManager : BaseVisionManager {
     @JvmStatic
     @JvmOverloads
     fun create(videoSource: VideoSource = Camera2VideoSourceImpl(application)) {
-        delegate = DelegateVisionManager.Impl()
+        mapboxTelemetry = MapboxTelemetry(
+            application,
+            mapboxToken,
+            MAPBOX_TELEMETRY_USER_AGENT
+        )
+
+        if (!isTurnstileEventSent) {
+            mapboxTelemetry.push(
+                    AppUserTurnstile(
+                            MAPBOX_VISION_IDENTIFIER,
+                            BuildConfig.VERSION_NAME
+                    )
+            )
+            isTurnstileEventSent = true
+        }
+
+        telemetryImageSaver = TelemetryImageSaverImpl()
 
         performanceProvider = PerformanceProvider.Impl(application).addTo(attachableModules)
 
         nativeVisionManager = NativeVisionManager(
```

Comment:
what will happen if we init `nativeVisionManager = NativeVisionManager(...)` -> `nativeVisionManager.destroy()` ->  how to _recreate_ `NativeVisionManager`